internal/sirius/payment.go

```diff
@@ -10,6 +10,7 @@ type Payment struct {
 	Source      string     `json:"source"`
 	Amount      int        `json:"amount"`
 	PaymentDate DateString `json:"paymentDate"`
+	Case        *Case      `json:"case,omitempty"`
```

Comment:
I'm still not very confident about pointers in Go, but we use direct references elsewhere so I'd assume we should do the same here:
```suggestion
	Case        Case       `json:"case,omitempty"`
```

Reply:
Thanks i'm not that knowledgable on pointers but `omitempty` doesn't work with structs as they don't have any [empty value](https://www.sohamkamani.com/golang/omitempty/) - and the article uses pointers as a solution to the problem

Reply:
Oooo yes, very clever 👍🏽